src/Lean/Elab/Syntax.lean

```diff
@@ -359,30 +365,30 @@ def inferMacroRulesAltKind : Syntax → CommandElabM SyntaxNodeKind
     pure quoted.getKind
   | _ => throwUnsupportedSyntax
 
-def elabNoKindMacroRulesAux (attrKind : Syntax) (alts : Array Syntax) : CommandElabM Syntax := do
+def expandNoKindMacroRulesAux (alts : Array Syntax) (cmdName : String) (mkCmd : Option Name → Array Syntax → CommandElabM Syntax) : CommandElabM Syntax := do
```

Comment:
Please add a comment for this function, and explain the purpose of `mkCmd`

Reply:
I am guessing it should be `private`.

Reply:
done